wgsl/index.bs

```diff
@@ -1039,15 +1039,42 @@ switch_stmt
   : SWITCH paren_rhs_stmt BRACKET_LEFT switch_body+ BRACKET_RIGHT
 
 switch_body
-  : CASE const_literal COLON BRACKET_LEFT case_body BRACKET_RIGHT
+  : CASE case_selectors COLON BRACKET_LEFT case_body BRACKET_RIGHT
   | DEFAULT COLON BRACKET_LEFT case_body BRACKET_RIGHT
 
+case_selectors
+  : const_literal (COMMA const_literal)*
+
 case_body
   :
   | statement case_body
   | FALLTHROUGH SEMICOLON
 </pre>
 
+A switch statement transfers control to one of a set of case clauses, or to a default clause,
+depending the evaluation of a selector expression of a scalar integer type.
+
+If the selector value equals a value in a case selector list, then control is transferred to
+the body of that case clause.
+If the selector value does not equal any of the case selector values, then control is
+transferred to the `default` clause.
+
+TODO(dneto): Is the default clause required? That would avoid undefined behaviour.
+
+A literal value must not appear more than once in the case selectors for a switch statement.
+
+Note: A literal value must not be duplicated within a case clause, and must not appear in more
+than one case selector list.
+
+Note: The value of the literal is what matters, not the spelling. For example `0` and `00` both
+denote the zero value.
+
+When control reaches the end of a case body, control normally transfers to the first statement
+after the switch statement.
+Alternately, executing a `fallthrough` statement transfers control to the body of the next case clause or
+default clause, whichever appears next in the switch body.
+If there is no such clause, then `fallthrough` transfers control to the first statement after the switch statement.
```

Comment:
This feels like it should be an error?

Reply:
That could be a good choice.
I leaned slightly toward what I wrote to retain generality for an automatic generator.  But I don't have that use case particularly.

Reply:
I looked up what Swift says about fallthrough, and I read it as non-committal at best on this point.

https://docs.swift.org/swift-book/LanguageGuide/ControlFlow.html#ID140